backend/canisters/community/impl/src/updates/start_video_call.rs

```diff
@@ -26,6 +26,16 @@ fn start_video_call_impl(args: Args, state: &mut RuntimeState) -> Response {
         return NotAuthorized;
     };
 
+    match (args.call_type, channel.chat.is_public.value) {
```

Comment:
Do we need this check? We'd want to remove it eventually, right?

Reply:
It depends. At the moment for the sake of simplicity I am using the visibility of the group / channel to determine what kind of call is permitted. So in a public group / channel you can only start "Spaces". 

But come to think of it, this is already incomplete because you can start normal calls in public channels in private communities.